usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -19,6 +20,117 @@
+
+            Send POST request to Tendrl API ``APIURL/GlusterCreateCluster
+
+        .. test_result:: 1
+
+            Server should return response in JSON format:
+
+                {
+                  "job_id": job_id
+                }
+
+            Return code should be **202**
+                with data ``{"message": "Accepted"}``.
+
+        """
+    nodes = []
+    provisioner_ip = None
+    network = None
+    node_ids = []
+    first = True
+    for x in valid_nodes:
+        if "tendrl/server" not in x["tags"]:
+            if first:
+                network = "{}/22".format(
+                    x["networks"][network_interface]["ipv4"][0])
+                first = False
+            ips = x["networks"][network_interface]["ipv4"]
+            nodes.append({
+                "role": "glusterfs/node",
+                "ip": ips[0] if type(ips) == list else ips})
+            node_ids.append(x["node_id"])
+            if "provisioner/gluster" in x["tags"]:
+                provisioner_ip = ips[0] if type(ips) == list else ips
+    LOGGER.debug("node_ips: %s" % nodes)
+    LOGGER.debug("provisioner: %s" % provisioner_ip)
+    """@pylatest api/gluster.cluster_create
+        .. test_step:: 2
+
+        Check if there is at least one gluster node for cluster creation.
+
+        .. test_result:: 2
+
+        Test passes if there is at least one gluster node.
+        """
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+    pytest.check(
+        len(nodes) > 0,
+        "There have to be at least one gluster node."
+        "There are {}".format(len(valid_nodes)))
+    job_id = api.create_cluster(
+        cluster_name,
+        str(uuid.uuid4()),
+        nodes,
+        provisioner_ip,
+        network)["job_id"]
+
+    api.wait_for_job_status(job_id)
+
+    integration_id = api.get_job_attribute(
+        job_id=job_id,
+        attribute="TendrlContext.integration_id",
+        section="parameters")
+    LOGGER.debug("integration_id: %s" % integration_id)
+
+    api.get_cluster_list()
+    # TODO(fbalak) remove this sleep after
+    #              https://github.com/Tendrl/api/issues/159 is resolved.
+    import time
+    time.sleep(30)
+
+    imported_clusters = [x for x in api.get_cluster_list()
+                         if x["integration_id"] == integration_id]
+    pytest.check(
+        len(imported_clusters) == 1,
+        "Job list integration_id '{}' should be "
+        "present in cluster list.".format(integration_id))
+
+    pytest.check(
+        len(imported_clusters[0]["nodes"]) == len(nodes),
+        "In cluster should be the same amount of hosts"
+        "(is {}) as is in API call for cluster creation."
+        "(is {})".format(len(imported_clusters["nodes"]), len(nodes)))
+
+    pytest.check(
+        set(node_ids) == set(imported_clusters[0]["nodes"].keys()),
+        "There should be imported these nodes: {}"
+        "There are: {}".format(node_ids, imported_clusters[0]["nodes"].keys()))
```

Comment:
You could have created a variable for `imported_clusters[0]["nodes"].keys()` as you are using this statement multiple times, improving readability a bit.